app/instance-initializers/sw.js

```diff
@@ -18,7 +18,13 @@ export function initialize(appInstance) {
 		if (isEnabled) {
 			swService.register(swDestFile);
 		} else {
-			swService.unregisterAll();
+			swService.unregisterAll().then(() => {
+				const purgeOnDisable = getWithDefault(config, 'ember-cli-workbox.purgeOnDisable', false);
```

Comment:
The `false` is not neccessary here, if the variable is not defined then the condicion will be falsy:

```js
const purgeOnDisable = get(config, 'ember-cli-workbox.purgeOnDisable');
```

Reply:
Done, thank you!